jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -61,3 +61,18 @@ set -e
 
 docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-log/vagrant.log &
 popd
+
+mountpath="$WORKSPACE/RackHD/docker/files/mount"
+retrytimes=5
+while [ ! -d "$mountpath" ]
+do
+    echo "mount is not finished"
+    retrytimes=$(($retrytimes-1))
+    echo "retry : $retrytimes times"
+    sleep 10
+    if [ $retrytimes -eq 0 ]; then
+        break
+    fi
+done
+echo "change the authority of mount"
+echo $SUDO_PASSWORD |sudo -S chown -R $USER:$USER $WORKSPACE/RackHD
```

Comment:
I think this steps ,in previous time we did in cleanup.sh , we need make this step more earlier , so I changed this to this script , so  why we don't keep consistent with cleanup.sh ? 

BTW: I've other comments in the PR, How about to take a quick glance , maybe some still make a bit sense ?
you mean the better way to judgement the mount operation ?  I want to ask first , why if [! -d $mountpath ]  is bad ?  and I really did find any better  way :) ,if you have ,please tell me .thanks .